### PR TITLE
Sqs working class

### PR DESCRIPTION
**Pull request by novichkovv**
q.class interface stays the same

### CODE REVIEW

File: protected/classes/aws_class.php [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            if(!empty($res->get('MessageId'))) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    public function get($num = 1, $delete = true)
+    {
+        if($num > 10) {
```
Comment (mosheka) [inline protected/classes/aws_class.php]: why we need this one?

Comment (novichkovv) [reply protected/classes/aws_class.php]: It's aws 10 records limit "By default, only one message will be returned. If you want to get more messages, make sure to use the MaxNumberOfMessages parameter and specify a number of messages (1 to 10)."

Comment (novichkovv) [reply protected/classes/aws_class.php]: config was added to /protected/config.php

Comment (mosheka) [reply protected/classes/aws_class.php]: So it means that we need to delete the current config.php in the /sqs folder

**Pull request merged**